resources/migrations/000_migrations.yaml

```diff
@@ -6212,3 +6212,79 @@ databaseChangeLog:
       changes:
         - dropTable:
             tableName: query_queryexecution
+
+# Create the native query snippets table, intended to store snippets and their metadata
+  - changeSet:
+      id: 163
```

Comment:
Next time you add a migration please announce it in the #migrations channel so we don't end up trying to use the same numbers